### ISSUE TITLE
Add Web Development Courses

### ISSUE THREAD

**Issue by ishandeveloper** [ishandeveloper/Coursify-hacktoberfest#1]
### Follow the guidelines in 'How To Contribute' section of README.md
and add a course that you've taken before or would like to recommend to others, related to web development.

**Comment by XIZT07** [comment]
Can I contribute to it?

**Comment by alphaCoder-69** [comment]
Can I work on it??

**Comment by ishandeveloper** [comment]
Yes @XIZT07 @alphaCoder-69 , you can!

There's no need to assign the issue, I'll keep these issues open for other contributors.
### Go ahead and make a PR ✨. Goodluck!

**Comment by saptarshibasu15** [comment]
Can I add a course even it is someone else's course on youtube?

**Comment by ishandeveloper** [comment]
@saptarshibasu15 Yes, definitely you can! 👍 

@fatmab2809 You can add courses/blogs/tutorials/articles/ebooks anything from any website, as long as it's 'free' and helpful for other people in the community 😃

**Comment by abraaoduarte** [comment]
I added a course in portuguese #36

**Comment by ashandilya** [comment]
Can I add courses in Hindi and English??

**Comment by ishandeveloper** [comment]
> 
> 
> Can I add courses in Hindi and English??

@ashandilya Yes, definitely you can! Just please remember to add a [Language-Name] tag, if you're adding a course in a language different from English.

For example, if you're planning to add a Hindi course, please add a 'Hindi' tag in the tags array.

**Comment by dindatiwi** [comment]
may i contribute here?
thanks

**Comment by GoyalRohan** [comment]
@ishandeveloper 
I would like to work on this issue
Can you please assign it to me?

**Comment by ishaanmane** [comment]
I would like to contribute to it
